SimG4Core/Generators/src/Generator.cc

```diff
@@ -85,14 +87,15 @@ Generator::Generator(const ParameterSet & p) :
 			    / ( 2*exp(-theEtaCutForHector) ) );
 
   edm::LogVerbatim("SimG4CoreGenerator") 
-    << " Rdecaycut= " << theRDecLenCut/cm 
+    << "SimG4Core/Generator: Rdecaycut= " << theRDecLenCut/cm 
     << " cm;  Zdecaycut= " << theDecLenCut/cm 
     << "Z_min= " << Z_lmin/cm << " cm; Z_max= " << Z_lmax 
     << " cm;  Z_hector = " << Z_hector << " cm\n"
-    << "ApplyPCuts: " << fPCuts << "  ApplyPtransCut: " << fPtransCut
-    << "  ApplyEtaCuts: " << fEtaCuts 
-    << "  ApplyPhiCuts: " << fPhiCuts
-    << "  ApplyLumiMonitorCuts: " << lumi;
+    << "                     ApplyCuts: " << fFiductialCuts
```

Comment:
@civanch trivial issue, why this space?

Reply:
@fabiocos , it is formatting a new line to have clear printout.